tests/test_integration.py

```diff
@@ -294,6 +297,60 @@ def test_send_many_traces(self):
 
         self._send_traces_and_check(traces, 2)
 
+    def test_send_single_trace_max_payload(self):
+        payload = Payload()
+
+        # compute number of spans to create to surpass max payload
+        trace = [Span(self.tracer, "child.span")]
+        trace_size = len(payload.encoder.encode_trace(trace))
+        num_spans = int(math.floor(payload.max_payload_size / trace_size))
```

Comment:
I wonder if there is a better way to test this, like set the payload max size to like 1b or something super small?

Reply:
This solution was copied over from `test_payload.py`:

https://github.com/DataDog/dd-trace-py/blob/master/tests/test_payload.py#L87-L111

Reply:
Though I agree we can find a better way to perform such a test.